src/main/java/dinkplugin/message/NotificationType.java

```diff
@@ -32,7 +32,8 @@ public enum NotificationType {
     LOGOUT("Player Logout", "logout.png", WIKI_IMG_BASE_URL + "Prop_sword.png"),
     TRADE("Player Trade", "trade.png", WIKI_IMG_BASE_URL + "Inventory.png"),
     CHAT("Chat Notification", "chat.png", WIKI_IMG_BASE_URL + "Toggle_Chat_effects.png"),
-    XP_MILESTONE("XP Milestone", "xpImage.png", WIKI_IMG_BASE_URL + "Lamp.png");
+    XP_MILESTONE("XP Milestone", "xpImage.png", WIKI_IMG_BASE_URL + "Lamp.png"),
+    EXTERNAL_PLUGIN("External Plugin", "externalImage.png", "https://runelite.net/img/logo.png");
```

Comment:
```suggestion
    EXTERNAL_PLUGIN("External Plugin", "externalImage.png", "https://runelite.net/img/plugin-hub/missingicon.png");
```

possible alternate icon - @Felanbird lmk which u prefer (current one felt a lil large)

Reply:
maybe the hub version of the plug? i'm not sure where the resource is for it, i hope in the resources folder on rl/rl

Reply:
that'd be nice, but isn't straightforward e.g. https://repo.runelite.net/plugins/icon/dink_x3-EnUbFOMb1DROmnMIT9yTqdIbxrwlnnYUENYxnnTc.png

Reply:
I mean the plug the top of the 🔧 tab, I assume(hope) that's bundled in runelite/runelite, not loaded externally
![image](https://github.com/user-attachments/assets/1b2131ef-0ee4-4983-9d92-19cf0b71429e)

Reply:
ah i can't read - good thinking, i like that better too